### ISSUE TITLE
code signing: iOS/Xcode 14.2

### ISSUE THREAD

**Issue by halmueller** [TreeMama/iSeaTree-React-Prototype#390]
The frameworks for gRPC-C++-gRPCCertificates-Cpp and React-Core-AccessibilityResources both fail to build, throwing "signing identity required".

Manually setting one allows the App Store build to finish. But this fix will be overwritten the next time Cocoapods is updated.

This is a known problem with Cocoapods and Xcode 14. The first link in the list below is to the discussion and fix. The fix version (1.12) has not yet landed in Homebrew.

Task: rebuild Cocoapods with v 1.12, test/validate our App Store build.

* [Xcode 14 build failed with manual code sign and app resource bundles · Issue #11402 · CocoaPods/CocoaPods](https://github.com/CocoaPods/CocoaPods/issues/11402)
* [swift - iOS Catalyst Cocoapod framework error - signing requires a development team - Stack Overflow](https://stackoverflow.com/questions/58547010/ios-catalyst-cocoapod-framework-error-signing-requires-a-development-team)
* [Flutter IOS build error - "select a development team in the Signing & Capabilities editor." - Stack Overflow](https://stackoverflow.com/questions/73711671/flutter-ios-build-error-select-a-development-team-in-the-signing-capabiliti)
* [ios - CocoaPod config to solve XCode 14, iOS16 Requires a development team. Select a development team in the Signing & Capabilities editor - Stack Overflow](https://stackoverflow.com/questions/73811360/cocoapod-config-to-solve-xcode-14-ios16-requires-a-development-team-select-a-d)
* [ios - Xcode 14 needs selected Development Team for Pod Bundles - Stack Overflow](https://stackoverflow.com/questions/72561696/xcode-14-needs-selected-development-team-for-pod-bundles)
* [Xcode 14 - After `pod install`, need to assign development team to each and every Pod target · Issue #11534 · CocoaPods/CocoaPods](https://github.com/CocoaPods/CocoaPods/issues/11534)
* [Requires a development team. Sele… | Apple Developer Forums](https://developer.apple.com/forums/thread/698052)
